File: fsharp/pangram/Pangram.fs
module Pangram
let alphabet = "abcdefghijklmnopqrstuvwxyz"
let isPangram (input: string): bool =
    let numUniqueCharacters =
        input.ToLower()
        |> Seq.filter alphabet.Contains
        |> Seq.distinct
        |> Seq.length
    numUniqueCharacters = alphabet.Length

File: haskell/pangram/src/Pangram.hs
module Pangram (isPangram) where
import Data.Char (isLetter, toLower)
import qualified Data.Set as Set

alphaSet :: Set.Set Char
alphaSet = Set.fromList ['a'..'z']

isPangram :: String -> Bool
isPangram text = alphaSet == Set.fromList (map toLower (filter isLetter text))